app/assets/javascripts/local_authorities.js

```diff
@@ -0,0 +1,15 @@
+document.addEventListener("DOMContentLoaded", function(event) {
+  var links_download_button = document.getElementById('links_download_button');
+  var href = links_download_button.href;
+
+  links_download_button.addEventListener('mouseenter', function() {
```

Comment:
I wonder if it's better to change the href each time a checkbox gets checked/unchecked, because I'm not sure if this event fires say if a user uses a keyboard to navigate to the button.

Reply:
My initial thought was the same and I started implementing it that way, but talking with @edwardkerry I decided to do it this way so to add it only once instead of once for each checkbox.

That said, I see your point, and I think it a stronger one. I'll amend this now.

Reply:
Could we add a second eventListener for `onfocus` to catch that, which I think would capture keyboard navigation? It is a small and quick script but might be good to limit how often it runs if possible

Reply:
Thinking about it, I feel that, semantically, it's more accurate to add event listeners to the checkboxes. Checking/unchecking a checkbox (not clicking the button) is what changes the outcome.

Reply:
Sounds sensible :+1: